src/app/assets/languages/en.json

```diff
@@ -201,6 +201,8 @@
           "backups-disabled-description": "Backups are available only when you configure {{brandname}} to perform cross-site replication. Remove the capability or enable cross-site replication.",
           "secured-disabled": "The server is not secured",
           "secured-disabled-description": "You must enable authentication and authorization in {{brandname}} Server to configure security. Remove the capability or enable security.",
+          "indexed-disabled": "This cache can not be indexed",
+          "indexed-disabled-description": "Indexing is only when Protobuf Schemas are added and encoding is not text/plain",
```

Comment:
The error is not accurate.

We should have 2 separate errors.

1) When the encoding is not Protobuf
2) When there aren't any protobuf types or schemas in the server.

1. The encoding for indexed caches must be application/x-protostream and the selected encoding is < display the encoding name the user has choses in the previous step>
2. You must register a protobuf schema containing indexed protobuf entities to configure indexing in this cache.